mobius/src/transformable/motif.rs

```diff
@@ -18,10 +20,25 @@ impl<T> Motif<T> {
         self.parts.iter()
     }
 
-    pub fn union(a: Self, b: Self) -> Self {
-        let parts: Vec<(T, usize)> = a.parts.into_iter().chain(b.parts).collect();
+    pub fn union(motifs: Vec<Self>) -> Self {
```

Comment:
This feels like a semigroup concat. Though I'll do that as a separate PR, I've added this to my notes